dtde/clusterlib.py

```diff
@@ -138,9 +135,44 @@ def generate_cluster_start_task(
         user_identity=POOL_ADMIN_USER_IDENTITY,
         wait_for_success=True)
 
+def upload_custom_script_config(custom_scripts=None):
+    with open(os.path.join(constants.CUSTOM_SCRIPTS_DEST, 'custom-scripts.yaml'), 'w+') as f:
+        f.write(yaml.dump(custom_scripts, default_flow_style = False))
+
+
+def move_custom_scripts(custom_scripts=None):
+    if custom_scripts is None:
+        return
+    
+    os.mkdir(os.path.join(constants.ROOT_PATH, 'node_scripts', 'custom-scripts'))
```

Comment:
If the directory exists, do we need to clean it up before copying files over? Can files from a previous cluster create still linger?

Reply:
... Or is that what line 203 does? May be worth doing it here as a first step in case the process exists unexpectedly between lines 197 and 203 since it shouldn't harm anything and provides an extra guarantee.

Reply:
I think we should actually clean up before we copy, and after. This will remove lingering files from a previous create, copy over the expected files, upload them and then delete them.

The only way someone could have lingering files, though, is if the program crashed between 197 and 203, since 203 does remove all the files.